src/resources/js/views/rsvp.js

```diff
@@ -19,20 +13,20 @@
 	 * @param obj button The dom object of the clicked button
 	 * @return void
 	 */
-	function tribe_rsvp_toggle_actions( button ) {
+	function tribe_rsvp_toggle_actions( $button ) {
```

Comment:
I'd like to avoid us going with "global" methods, and start saving stuff on to an object to allow external use if needed.